configs/BROTHERHOBBYH743/config.h

```diff
@@ -0,0 +1,168 @@
+/*
+ * This file is part of Betaflight.
+ *
+ * Betaflight is free software. You can redistribute this software
+ * and/or modify this software under the terms of the GNU General
+ * Public License as published by the Free Software Foundation,
+ * either version 3 of the License, or (at your option) any later
+ * version.
+ *
+ * Betaflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define FC_TARGET_MCU                       STM32H743
+
+#define BOARD_NAME                          BROTHERHOBBYH743
+#define MANUFACTURER_ID                     BROT
+
+// LEDS
+#define BEEPER_PIN                          PA15
+#define BEEPER_INVERTED
+#define LED_STRIP_PIN                       PA8
+#define LED0_PIN                            PE9
+#define LED1_PIN                            PA7
+
+// MOTORS & SERVOS
+#define MOTOR1_PIN                          PB0
+#define MOTOR2_PIN                          PB1
+#define MOTOR3_PIN                          PA0
+#define MOTOR4_PIN                          PA1
+#define MOTOR5_PIN                          PA2
+#define MOTOR6_PIN                          PA3
+#define MOTOR7_PIN                          PD12
+#define MOTOR8_PIN                          PD13
+
+#define SERVO1_PIN                          PD14
+#define SERVO2_PIN                          PD15
+#define SERVO3_PIN                          PE5
+#define SERVO4_PIN                          PE6
```

Comment:
Servos 1 and 2 are on TIM4 and conflict with motors 7 & 8. 
```suggestion
#define SERVO1_PIN                          PE5
#define SERVO2_PIN                          PE6
```